tools/cmake/Makefile

```diff
@@ -8,12 +8,13 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=cmake
 PKG_VERSION:=3.24.1
+PKG_VERSION_MAJOR:=$(word 1,$(subst ., ,$(PKG_VERSION))).$(word 2,$(subst ., ,$(PKG_VERSION)))
```

Comment:
This is wonderful. A lot of packages in the packages feed can take advantage of this.

Reply:
Wonder... should we make this an helper?

Reply:
Probably not.